### ISSUE TITLE
Attribute calculation

### ISSUE THREAD

**Issue by jhoetter** [code-kern-ai/refinery#40]
**Is your feature request related to a problem? Please describe.**
Similar to pandas, I want to be able to create new attributes given some logic to apply.

**Describe the solution you'd like**
This could look very similar to labeling functions, e.g. as follows:
```python
def cat_attributes(record):
    return str(record["attr1"]) + str(record["attr2"])
```
to concatenate two textual attributes.

To avoid data clutter, this should also allow users to a) distinguish between original attributes in the data and b) delete attributes

**Describe alternatives you've considered**
Changing the data before you upload it - very static and not dev friendly

**Additional context**
I already implemented a proof-of-concept before our OS release :)

**Comment by jhoetter** [comment]
related to issue #14

**Comment by jhoetter** [comment]
We could integrate a **modifiers** section in the project settings page, where users see what attribute calculations they have (similar to a pandas DataFrame):
![project-settings](https://user-images.githubusercontent.com/62521561/182246783-86e8f762-2bf6-45d1-8a2f-131ce4f71aeb.png)

We could "reuse" what we have for the labeling functions IDE:
![modifier-ide](https://user-images.githubusercontent.com/62521561/182246794-0ad64693-5bd3-4253-aed1-6594d2609b6a.png)

**Comment by FelixKirsch** [comment]
implemented in release v1.3.0
